packages/sdks/javascript/Makefile

```
@@ -0,0 +1,7 @@
+test:
```

1. Please choose one, and only one, naming convention between for `kebab-case` or  `snake_case` and please don't mix `_` and `-` in the same image name.

```suggestion
IMAGE=devopness-sdk-js
TAG ?= latest

.PHONY: all build

all: build

test:
```

---

@jfoliveira resolved